active/tab/tabcontrol.md

```diff
@@ -0,0 +1,245 @@
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+## To replicate the behavior of Microsoft Edge
+
+``` xml
+<TabControl TabWidthBehavior="Equal"
+            CanCloseTabs="True"
+            CloseButtonOverlay="OnHover"
+            CanDragItems="True"
+            CanReorderItems="True"
+            TabDraggedOutside="OpenTabInNewWindow">
+    <TabControl.TabFooter>
+        <Button Content="+" Click="NewTab_Click" />
+    </TabControl.TabFooter>
+    ...
+</TabControl>
+```
+
+## Put tabs in the titlebar area
+
+![A sample app that shows the Tab control can extend into the titlebar area](images/tab-extend-to-title.png)
+
+``` xml
+<TabControl>
+    <TabItem Header="Tab 1" />
+    <TabItem Header="Tab 2" />
+    <!-- More tabs -->
+</TabControl>
+
+<Grid x:Name="CustomDragRegion" Width="200" Height="40" HorizontalAlignment="Right" VerticalAlignment="Top" />
+```
+
+``` csharp
+// Customize the titlebar area using the guidance from here: https://docs.microsoft.com/en-us/windows/uwp/design/shell/title-bar
+public MainPage()
+{
+    this.InitializeComponent();
+
+    var coreTitleBar = CoreApplication.GetCurrentView().TitleBar;
+    coreTitleBar.ExtendViewIntoTitleBar = true;
+
+    Window.Current.SetTitleBar(CustomDragRegion);
+}
+```
+
+## Create a new window when tearing out tabs
+
+![An example picture showing that a tab can be torn out and moved to its own window.](images/tab-new-window.png)
+
+See the [TabView tear out sample](https://github.com/windows-toolkit/Sample-TabView-TearOff/tree/master/TabViewTear) for a more complete sample.
+
+``` xml
+<TabControl CanDragItems="True"
+            CanReorderItems="True"
+            TabDraggedOutside="TabControl_TabDraggedOutside">
+```
+``` csharp
+// NOTE: The app is responsible for writing this code. We will provide a sample that may look something like:
+private async void TabControl_TabDraggedOutside(object sender, TabDraggedOutsideEventArgs e)
+{
+    // Create a new AppWindow
+    AppWindow newWindow = await AppWindow.TryCreateAsync();
+
+    // Create the content for the new window
+    var newPage = new MainPage();
+
+    // Remove tab from existing list
+    Tabs.Items.Remove(e.Tab);
+
+    // Add tab to list of Tabs on new page
+    newPage.AddItemToTabs(e.Tab);
+
+    // Set the Window's content to the new page
+    ElementCompositionPreview.SetAppWindowContent(newWindow, newPage);
+
+    // Show the window
+    await newWindow.TryShowAsync();
+}
+```
+
+## Databind to a set of tabs 
+
+``` xml
+<TabControl ItemsSource="{x:Bind TabItemCollection}" />
+```
+
+# Remarks
```

Comment:
I would be interested in learning more about TabControl's resizing behavior. The API includes TabWidthBehavior, which is part of this story. I'm also curious about what happens to the space allocated for TabHeader, TabFooter, and TabActionContent as the tab content grows or window width shrinks.

What if I put a CommandBar with dynamic overflow into the TabFooter, how will the elements behave? What if I put a stack panel with multiple items in the TabFooter? Do I need to wrap it in a ScrollViewer to ensure all the items remain accessible?